code/modules/reagents/chemistry/recipes/special.dm

```diff
@@ -162,10 +171,27 @@ GLOBAL_LIST_INIT(food_reagents, build_reagents_to_food()) //reagentid = related
 			return food_reagent_ids
 	return ..()
 
+///Random recipey for meme chem metalgen. Always requires wittel and resets every 3 days
+/datum/chemical_reaction/randomized/metalgen
+	persistent = TRUE
+	persistence_period = 3 //Resets every three days. It's the ultimate meme and is best not worn out
+	randomize_req_temperature = TRUE
+	possible_catalysts = list(/datum/reagent/wittel)
```

Comment:
You probably want to set minimum catalysts to 1 here or it can randomly roll none.

Reply:
Ooh, thanks, I'll do that